code/modules/mob/living/carbon/human/human.dm

```diff
@@ -997,6 +997,7 @@
 
 /mob/living/carbon/human/species/lizard
 	race = /datum/species/lizard
+	deathsound = 'sound/voice/lizard/deathsound.ogg'

```

Comment:
Make this sort of thing an On species gain/lose sort of thing. That'll sort out all of the sounds

Reply:
I ended up figuring this out and fixing it up on my own before seeing this, but thank you for your help regardless!

Reply:
Yee yee. Does it still apply the correct sound if you swap species?